unstructured/documents/elements.py

```diff
@@ -221,32 +285,25 @@ def convert_coordinates_to_new_system(
     ) -> Optional[Tuple[Tuple[Union[int, float], Union[int, float]], ...]]:
         """Converts the element location coordinates to a new coordinate system. If inplace is true,
         changes the coordinates in place and updates the coordinate system."""
-        if self._coordinate_system is None or self.coordinates is None:
+        if (
+            not self.metadata.coordinates
+            or self.metadata.coordinates.points is None
+            or self.metadata.coordinates.system is None
+        ):
```

Comment:
If one of `points` or `system` is populated but the other is not, we might want to raise an error (maybe `ValueError`) to a) flag a user that both (or neither) are needed and b) avoid failing silently when something unexpected has happened in the code and we've only gotten one of the expected parameters.

Reply:
I updated the `CoordinatesMetadata` constructor to raise this error when it's called with incomplete data:
https://github.com/Unstructured-IO/unstructured/pull/827/commits/123466997719361ff971d895b0b8596f9b813527